File: java/252-MeetingRooms.java
public class Solution {
    public boolean canAttendMeetings(List<Interval> intervals) {
        
       Collections.sort(intervals, (a, b) -> a.start - b.start);

       for(int i=0; i+1 < intervals.size(); i++){
           if(intervals.get(i).end > intervals.get(i+1).start){
               return false;
           }
       }
       return true;
    }
}

File: csharp/252-Meeting-Rooms.cs
public class Solution {
    public bool CanAttendMeetings(int[][] intervals) {
        
            Array.Sort(intervals, (a, b) => a[0] < b[0] ? -1 : 1);
            for (int i = 1; i < intervals.Length; i++)
            {
                if (intervals[i][0] < intervals[i - 1][1])
                {
                    return false;
                }
            }
            return true;
    }
}
